kpops/components/base_components/kafka_connector.py

```diff
@@ -98,7 +100,16 @@ def helm(self) -> Helm:
         )
         return helm
 
-    def _get_resetter_helm_chart(self) -> str:
+    @property
+    def _connector_resetter_release_name(self) -> str:
```

Comment:
wdyt about shortened name without connector? would be analogous to `_resetter_helm_chart` property